com.github.itprojects.MasVisGtk.json

```diff
@@ -0,0 +1,99 @@
+{
+    "id" : "com.github.itprojects.MasVisGtk",
```

Comment:
the document clearly say that `io.github` should be used in that case.

Reply:
Please follow the docs, not what other apps are doing. Those apps were submitted before the policy.

Reply:
So policies _CAN_ change.

Reply:
A new code version exists (compiles and runs latest deps), but it seems it will have to be refactored.

Reply:
> So policies _CAN_ change.

Yes